app/models/reservation_scopes.rb

```diff
@@ -56,32 +42,29 @@ def self.included(base) # rubocop:disable MethodLength, AbcSize
       scope :ends_on_days, lambda { |start_date, end_date|
         where(due_date: start_date..end_date)
       }
-      scope :reserved_on_date, ->(date) { overlaps_with_date(date).finalized }
+      scope :reserved_on_date, lambda { |date|
+        overlaps_with_date(date).not_available
+      }
       scope :for_eq_model, lambda { |eq_model|
-        where(equipment_model_id: eq_model.id).finalized
+        where(equipment_model_id: eq_model.id)
       }
       scope :active_or_requested, lambda {
-        where('checked_in IS NULL and approval_status != ?', 'denied').recent
+        where(status: Reservation.statuses.values_at(
+          *%w(requested reserved checked_out)).compact)
       }
       scope :notes_unsent, ->() { where(notes_unsent: true) }
-      scope :requested, lambda {
-        where('start_date >= ? and approval_status = ?',
-              Time.zone.today, 'requested').recent
-      }
+
       scope :approved_requests, lambda {
-        where('approval_status = ?', 'approved').recent
-      }
-      scope :denied_requests, lambda {
-        where('approval_status = ?', 'denied').recent
+        where('flags & ? > 0', Reservation::FLAGS[:request]).reserved
       }
+
       scope :missed_requests, lambda {
-        where('approval_status = ? and start_date < ?', 'requested',
-              Time.zone.today).recent
+        where('start_date < ?', Time.zone.today).requested
       }
       scope :for_reserver, ->(reserver) { where(reserver_id: reserver) }
       scope :reserved_in_date_range, lambda { |start_date, end_date|
         where('start_date <= ? and due_date >= ?', end_date, start_date)
-          .finalized
+          .reserved
```

Comment:
I'm assuming that we never call this in a situation where we want to also return checked out / otherwise processed reservations (e.g. we're always looking into the future)? This is so close to `overlaps_with_date`, I'm wondering if we should refactor `overlaps_with_date` to just be `reserved_in_date_range(date, date)`. Thoughts?